index.html

```diff
@@ -18,13 +18,18 @@
             <header>
                 <nav class="navbar">
                     <a class="logo" href="">Hamdullah</a>
-                    <img src="images/Union.png" alt="">
+                    <img class="logo-image" src="images/Union.png" alt="">
+                    <div class="menu-bar">
+                        <a class="portfolio-menu" href="">Portfolio</a>
+                        <a class="about-menu" href="">About</a>
+                        <a class="contact-menu" href="">Contact</a>
+                    </div>
                 </nav>
             </header>
 
             
            <!-- The main section  -->
-            <main>
+            <main class="main-class">
                 <section class="hero">
 
                 <h1 class="heading">I’m Hami
```

Comment:
- [ ] Kindly update your headline section for desktop version to make it look as much as possible like Figma. The `width` and text `color` are not the same.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194668867-5265cedf-bdaf-4c52-a5d6-e036b9c296d3.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194668882-222b6f82-5e7f-4918-8ea2-a795d016bc21.png)